phalcon/assets/collection.zep

```diff
@@ -49,7 +49,7 @@ class Collection implements \Countable, \Iterator
 
 	protected _attributes = [] { get };
 
-	protected _join = true { get };
+	protected _join = false { get };
```

Comment:
Well, your arguments are reasonable. Could you please update changelog, remove comments form PR?